### ISSUE TITLE
Flatten the Document Structure to Make the Page Load Faster

### ISSUE THREAD

**Issue by aj-stein-nist** [usnistgov/OSCAL#1369]
### User Story

As an OSCAL tool developer, in order to quickly and conveniently review reference documentation for OSCAL models, I would like the pages to load quickly (with a better HTML document model that loads more efficiently).

### Goals

- [x] Design simplified HTML document model as advised by usnistgov/OSCAL#1334 as flattened as possible
- [ ] Implement it in the reference document page generation

### Acceptance Criteria

- [ ] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [ ] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [ ] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

**Comment by aj-stein-nist** [comment]
As Dave has rolled of the project, I will re-assign this to myself in the interim until I can update it.

**Comment by aj-stein-nist** [comment]
Talked with @david-waltermire-nist and @nikitawootten-nist on Friday. Dave still likely has some WIP code for this issue on his workstation he can get over to us on Tuesday.

**Comment by nikitawootten-nist** [comment]
This issue is blocked until we can verify performance via #1373

**Comment by aj-stein-nist** [comment]
AJ is to sync up with @nikitawootten-nist and @wendellpiez to understand #1373 and work on that as focus first with them to unblock this or determine it is out of scope from sprint and/or slide down priority.

**Comment by aj-stein-nist** [comment]
Throw this into backlog for now. Review from #1373 indicates gain from this would be negligible and it is the least of our concerns performance wise.

**Comment by aj-stein-nist** [comment]
@nikitawootten-nist can you and I discuss tomorrow and determine if this is a WONTFIX since this specific line of changes was determined to have negligible impact?

**Comment by nikitawootten-nist** [comment]
> @nikitawootten-nist can you and I discuss tomorrow and determine if this is a WONTFIX since this specific line of changes was determined to have negligible impact?

Sure. If I remember correctly @wendellpiez and I trialed a flatter design (https://github.com/usnistgov/OSCAL/issues/1373) and determined that it did not affect lighthouse metrics or perceived performance. The static generation of title headings had a much more pronounced affect.

**Comment by aj-stein-nist** [comment]
OK I will close this issue and we reopen if the situation changes.
